pkg/scheduler/ugm/queue_tracker.go

```diff
@@ -151,3 +151,23 @@ func (qt *QueueTracker) getResourceUsageDAOInfo(parentQueuePath string) *dao.Res
 func (qt *QueueTracker) getRunningApplications() map[string]bool {
 	return qt.runningApplications
 }
+
+func (qt *QueueTracker) cleanQueueTracker(queuePath string) {
+	idx := strings.Index(queuePath, configs.DOT)
+	childQueuePath := ""
+	if idx != -1 {
+		childQueuePath = queuePath[idx+1:]
+	}
+	childIndex := strings.Index(childQueuePath, configs.DOT)
+	immediateChildQueueName := childQueuePath
+	if childIndex != -1 {
+		immediateChildQueueName = childQueuePath[:childIndex]
+	}
```

Comment:
I just added a function `getChildQueuePath()` which returns two string, essentially this part can be replaced with that call.

Reply:
Good suggestion, addressed in latest PR.